### PR TITLE
[SDX-1073] Added DateRange component as part of ViewHeader

### PR DESCRIPTION
**Pull request by jenslonkowski**

### CODE REVIEW

**Comment by muthan** [comment]
the datepicker looks strange with only 6 days per row. columns also doesn't fit with headers.
screenshot from 2016-12-15 11-59-16

File: src/framework/viewHeader/dateRange/DateRange.jsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    });
+    let dateRange;
+    let dateRangeArrow;
+    let dateRangePicker;
+    let dateRangePickerStyle;
+
+    if (this.state.endDate && this.state.startDate) {
+      const formattedEndDate = this.state.endDate.format(this.DATE_FORMAT);
+      const formattedStartDate = this.state.startDate.format(this.DATE_FORMAT);
+      dateRange = `${formattedStartDate} - ${formattedEndDate}`;
```
Comment (ahollesen) [inline src/framework/viewHeader/dateRange/DateRange.jsx]: What about only showing one date if both are same?

File: src/framework/viewHeader/dateRange/DateRange.jsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    return (
+      <div className="dateRange">
+        <div
+          className={dateRangeRangeClasses}
+          onClick={this.onClick}
+          ref="dateRangeRange"
+        >
+          <span className="icon icon-calendar-white" />{dateRange}
```
Comment (ahollesen) [inline src/framework/viewHeader/dateRange/DateRange.jsx]: Line wrap.

File: src/framework/viewHeader/dateRange/_dateRange.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    height: 0;
+    margin: 3px auto -8px;
+    width: 0;
+  }
+
+  .dateRange__picker {
+    margin-top: 8px;
+    position: absolute;
+    width: 560px;
+    z-index: 1;
```
Comment (ahollesen) [inline src/framework/viewHeader/dateRange/_dateRange.scss]: Not sure if needed.

File: src/framework/viewHeader/dateRange/DateRange.jsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            DaySelected: {
+              background: '#3CCEA3',
+              color: '#FFFFFF',
+              fontWeight: 400,
+            },
+          }}
+        />
+      );
+      dateRangePickerStyle = {
+        [this.props.isDateRangePickerToRight ? 'left' : 'right']: 0,
```
Comment (ahollesen) [inline src/framework/viewHeader/dateRange/DateRange.jsx]: Would be `...ToTheRight`. But I'd just call it either `isRight` or `floatRight` or something like that.

File: src/framework/viewHeader/dateRange/_dateRange.scss (hunk 3 above)
Comment (jenslonkowski) [inline src/framework/viewHeader/dateRange/_dateRange.scss]: It's needed. Look at the Grid View Integration and remove it.

**Pull request merged**